Source/ShoppingLists/Main.lua

```diff
@@ -57,3 +65,13 @@ function Auctionator.ShoppingLists.ImportFromString(listName, importString)
 
   list.items = {strsplit("\n", importString)}
 end
+
+function Auctionator.ShoppingLists.Prune()
+  local indexShift = 0
```

Comment:
Instead of mutating, I would just regenerate the lists table:
```lua
  local lists = {}

  for _, list in ipairs(Auctionator.ShoppingLists.Lists) do
    if not list.isTemporary then
      table.insert(lists, list)
    end
  end

  Auctionator.ShoppingLists.Lists = lists
```
I think we also need to fire an event to get the shopping lists in our AH tab to re-render?

Reply:
Prune is only called on startup, and that's before any frames have been registered as the AH isn't open yet.

Reply:
This runs before the AH is opened, so we don't need to worry about updating any frames